addons/website_blog/data/website_blog_demo.xml

```diff
@@ -329,5 +336,68 @@ Naturally, as you grow in your love of astronomy, you will find many celestial b
 ]]>
             </field>
         </record>
+
+        <record id="blog_comment_1" model="mail.message">
+            <field name="body">Beautiful !, I know where will be my next trip !</field>
+            <field name="model">blog.post</field>
+            <field name="res_id" ref="blog_post_1"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="base.partner_admin"/>
```

Comment:
The author can't comment his own article :-) 
Make it comment by Joel Willis.
"Beautiful! I plan to go there next holidays."